src/pages/Calculator/Steps/SpeciesSelection/index.js

```diff
@@ -110,8 +112,16 @@ const SpeciesSelection = ({ completedStep, setCompletedStep }) => {
           flexDirection="column"
           p="1rem"
         >
+
           <SearchField handleChange={updateQuery} value={query} />
-          <Diversity diversitySelected={diversitySelected} />
+          {seedsSelected.length === 0
+            ? (
+              <Typography variant="p">
+                Click Show details to see species options, or use the search bar to find a specific species you can pair to create a mix.
```

Comment:
Maybe set this text to have same height as the Diversity bar to prevent layout shift when selecting crops